frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.pdf.component.js

```diff
@@ -1,19 +1,625 @@
+          top={courtTypeTop}
+          shouldShow={data.case.courtType === "Justice"}
+        />
+        <PositionedString
+          dataKey="case.courtAddress"
+          left="27.8%"
+          top="43.2%"
+        />
+
+        <PositionedString left="25.9%" top="40.15%" debugKey="judicialDistrict">
+          {data.case.courtAddress &&
+            getJudicialDistrict(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+
+        <PositionedString left="50.7%" top="40.15%" debugKey="county">
+          {data.case.courtAddress &&
+            getCounty(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+
+        <PositionedString debugKey="petitionerName" left="12.5%" top="50.2%">
+          {`${renderData("person.firstName")} ${renderData(
+            "person.middleName"
+          )} ${renderData("person.lastName")}`}
+        </PositionedString>
+
+        <PositionedString
+          dataKey="case.caseNumber"
+          left={caseJudgeLeft}
+          top="56%"
+        />
+
+        <PositionedString
+          dataKey="case.judgeName"
+          left={caseJudgeLeft}
+          top="60.5%"
+        />
+        <PositionedString
+          dataKey="case.commissionerName"
+          left={caseJudgeLeft}
+          top="65.3%"
+        />
+
+        <PositionedCheckmark dataKey="case.filingFee" left="17.8%" top="78%" />
+        <PositionedString
+          dataKey="case.filingFeeAmount"
+          top="80.2%"
+          left="31.7%"
+        />
+        <PositionedCheckmark dataKey="case.ocapFee" left="17.8%" top="82.6%" />
+        <PositionedCheckmark
+          dataKey="case.divorceEducation"
+          left="17.8%"
+          top="85.1%"
+        />
+
+        <PositionedCheckmark
+          dataKey="case.divorceOrientation"
+          left="56.2%"
+          top="78%"
+        />
+        <PositionedCheckmark
+          dataKey="case.vitalRecordFee"
+          left="56.2%"
+          top="82%"
+        />
+        <PositionedCheckmark
+          dataKey="case.serviceFee"
+          left="56.2%"
+          top="87.2%"
+        />
+      </RenderPage>
+
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-02.png">
+        <PositionedCheckmark
+          dataKey="case.appealFee"
+          left="17.8%"
+          top="13.4%"
+        />
+        <PositionedCheckmark
+          dataKey="case.appealFeeLarge"
+          left="20.8%"
+          top="15.6%"
+        />
+        <PositionedCheckmark
+          dataKey="case.appealFeeSmall"
+          left="20.8%"
+          top="17.2%"
+        />
+        <PositionedCheckmark
+          dataKey="case.otherFeeToWaive"
+          left="56.3%"
+          top="13.2%"
+        />
+        <PositionedString
+          debugKey="otherFeeToWaiveExplanation"
+          left="64.8%"
+          top="13.3%"
+        >
+          {`${renderData("case.otherFeeToWaiveName")} ( $${renderData(
```

Comment:
The `$${` is for a dollar sign before the break in the template string? Just making sure the `$$` is intentional

Reply:
Yes that dollar sign is intentional